Example/SnapshotTests/BPKNudgerSnapshotTest.m

```diff
@@ -0,0 +1,94 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018-2021 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#import <FBSnapshotTestCase/FBSnapshotTestCase.h>
+
+#import <Backpack/Backpack.h>
+
+#import "BPKSnapshotTest.h"
+
+NS_ASSUME_NONNULL_BEGIN
+@interface BPKNudgerSnapshotTest : FBSnapshotTestCase
+@end
+
+@implementation BPKNudgerSnapshotTest
+
+- (void)setUp {
+    [super setUp];
+    self.recordMode = NO;
+}
+
+- (UIView *)createNudgerWithValue:(double)value {
+    BPKNudgerConfiguration *nudgerConfiguration = [[BPKNudgerConfiguration alloc]
+        initWithLabelFormatter:^(double value) {
+            NSNumber *number = [NSNumber numberWithDouble:value];
+            return [NSNumberFormatter localizedStringFromNumber:number numberStyle:NSNumberFormatterDecimalStyle];
+        }
+        accessibilityLabelFormatter:^(double value) {
+            return [NSString stringWithFormat:@"%@ passengers", [NSNumber numberWithDouble:value]];
```

Comment:
Same comment as the REAMDE.md here but it doens't really matter since this string ins't visible in the test. Maybe just return an empty string

Reply:
Returning the same string and ignoring the value provided by the nudger fails to test that the nudger is giving us the correct value.

Reply:
But this is a snapshot test and the accessibility label isn't visible in the resulting screenshot. Thus, it doesn't matter what value it has?